ru_RU/readme.md

```diff
@@ -0,0 +1,778 @@
+
+test.cb.beforeEach(t => {
+	setTimeout(t.end);
+});
+
+test.afterEach.cb(t => {
+	setTimeout(t.end);
+});
+
+test.after(t => {
+   return new Promise(/* ... */);
+});
+```
+
+Хуки `beforeEach` и `afterEach` могут делить context с тестом:
+
+```js
+test.beforeEach(t => {
+	t.context.data = generateUniqueData();
+});
+
+test(t => {
+	t.is(t.context.data + 'bar', 'foobar');
+});
+```
+
+Context по умолчанию - объект, но также он может быть определен:
+
+```js
+test.beforeEach(t => {
+	t.context = 'unicorn';
+});
+
+test(t => {
+	t.is(t.context, 'unicorn');
+});
+```
+
+### Цепочка тестовых модификаторов
+
+Вы можете сцепить тестовые модификаторы вместе следующим образом:
+
+```js
+test.before.skip([title], testFn);
+test.skip.after(....);
+test.serial.only(...);
+test.only.serial(...);
+```
+
+Это особенно полезно, когда необходимо временно использовать `skip` и `only` для теста, без потери информативности и поведения других модификаторов.
+
+### Кастомный модуль сравнений
+
+Вы можете использовать любую библиотеку сравнений вместо или с тем функционалом, что встроен в AVA, но в этом случае Вы не сможете использовать метод `.plan()`, [пока что](https://github.com/sindresorhus/ava/issues/25).
+
+```js
+import assert from 'assert';
+
+test(t => {
+	assert(true);
+});
+```
+
+### <a name="ES2015"></a>Поддержка ES2015
+
+AVA имеет встроенную поддержку ES2015 через [Babel 6](https://babeljs.io). Просто пишите свои тесты в ES2015. Никакой дополнительной настройки не нужно. Вы можете использовать любую версию Babel в своем проекте. Мы используем наш собственный собранный Babel с [`es2015`](http://babeljs.io/docs/plugins/preset-es2015/) и [`stage-2`](http://babeljs.io/docs/plugins/preset-stage-2/) предустановками.
+
+#### Транспиллинг импортированных модулей
+
+AVA в данный момент транспиллит только запущенные тесты. *AVA не делает транспиллинг импортированных модулей ```import``` вне теста.* Есть веские причины для этого подхода, это может быть не тем, что Вы ожидали!
+
+Простым решением может быть [Babel's require hook](https://babeljs.io/docs/usage/require/) для транспиллинга импортированных модулей на лету. Потому как AVA поддерживает модульный синтаксис ES2015, Вы можете использовать это для импорта необходимого хука:
+
+```js
+import test from 'ava';
+import 'babel-core/register';
+import foo from './foo'; // <-- foo может быть написан в ES2015!
+
+test('foo bar', t => {
+	t.same('baz', foo('bar'));
+});
+```
+
+[#111](https://github.com/sindresorhus/ava/issues/111) взгляните на это как на возможное улучшение.
+
+### Поддержка обещаний
+
+Если Вы возвращаете обещание в свое тесте, то Вам не нужно явно завершать тест, так как он завершиться сразу после успешного выполнения обещания.
+
+```js
+test(t => {
+	return somePromise().then(result => {
+		t.is(result, 'unicorn');
+	});
+});
+```
+
+### Поддержка генераторов
+
+AVA имеет встроенную поддержку [генераторов](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*).
```

Comment:
Link `https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/function*` is better for russians